test.sh

```diff
@@ -12,7 +12,7 @@ CONTINUE_FROM="$1"
 
 if [[ -z $CONTINUE_FROM ]]; then
   start flake8
-  flake8 || die 'Try: autopep8 --in-place --aggressive -r .'
+  flake8 src || die 'Try: autopep8 --in-place --aggressive -r .'
```

Comment:
This is just a performance improvement, so it doesn't need to scan other directories that don't contain python?

Reply:
Indeed - I was using venv in the ivt directory and flake was trying to analyse that.